src/sentry/incidents/subscription_processor.py

```diff
@@ -758,6 +805,6 @@ def update_alert_rule_stats(alert_rule, subscription, last_update, alert_counts,
     pipeline.execute()
 
 
-def get_redis_client():
+def get_redis_client() -> Any:
```

Comment:
I believe the correct type to use here is `sentry.utils.redis.RetryingRedisCluster`